apps/meteor/client/startup/e2e.ts

```diff
@@ -38,26 +37,14 @@ Meteor.startup(() => {
 	let observable: Meteor.LiveQueryHandle | null = null;
 	let offClientMessageReceived: undefined | (() => void);
 	let offClientBeforeSendMessage: undefined | (() => void);
-	let unsubNotifyUser: undefined | (() => void);
 	Tracker.autorun(() => {
 		if (!e2e.isReady()) {
 			offClientMessageReceived?.();
-			unsubNotifyUser?.();
-			unsubNotifyUser = undefined;
 			observable?.stop();
 			offClientBeforeSendMessage?.();
 			return;
 		}
 
-		unsubNotifyUser = sdk.stream('notify-user', [`${Meteor.userId()}/e2ekeyRequest`], async (roomId, keyId): Promise<void> => {
```

Comment:
@KevLehman Why are we removing the old way of key exchange?

Reply:
IMO we need both ways of key exchange, asynchronous (new way) and synchronous (old way).